services/ui-src/src/components/cards/TemplateCard.tsx

```diff
@@ -67,7 +67,7 @@ export const TemplateCard = ({
               leftIcon={
                 <Image src={downloadIcon} alt="Download Icon" height="1.5rem" />
               }
-              isDisabled={isDisabled}
+              isDisabled={false}
```

Comment:
Question: Why is this one always false?

Reply:
Originally, the top-level / incoming `isDisabled` prop was set to `false` and only used for this particular download button, so I assumed users need to be able to download the template regardless of their access.